recipes/libcublas/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "libcublas" %}
+{% set version = "12.0.1.189" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 38272cc9184b4a57a6250fc2cd914ffc827f9f8cd5a3e1356bc3c31f843caf8f  # [linux64]
+  sha256: 2d46e735f8e76e2eb7df1b4f41d56aa8d5d2683f99988bac82d6ae33fb6d344b  # [ppc64le]
+  sha256: 96b68f6821684f39f338862639ce74a74c75830bf44dcd17ef372a1cb8c2304d  # [aarch64]
+  sha256: 88f29bec81880844da2eee13f1cdc51e1ccd8b4c60af6df633577810b30f80d0  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:
+    strong:
+      - __glibc >=2.17
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+  run:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+
+outputs:
+  - name: libcublas
+    files:
+      - lib/lib*blas*.so.*            # [linux]
+      - targets/*/lib/lib*blas*.so.*  # [linux]
+      - Library\bin                   # [win]
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcublas.so  # [unix]
+        - if not exist %LIBRARY_LIB%\\cublas.lib exit 1  # [win]
+
+  - name: libcublas-dev
+    requirements:
+      run:
+        - libcublas >={{ version }}
+    files:
+      - lib/lib*blas*.so            # [linux]
+      - targets/*/lib/lib*blas*.so  # [linux]
+      - lib/stubs                   # [linux]
+      - include                     # [linux]
+      - targets/*/include/*         # [linux]
+      - lib/pkgconfig               # [linux]
+      - src
+      - Library\include             # [win]
+      - Library\lib                 # [win]
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcublas.so  # [unix]
+        - if not exist %LIBRARY_LIB%\\cublas.lib exit 1  # [win]
+
+  - name: libcublas-static
+    build:
+      skip: True  # [not linux]
+    requirements:
+      run:
+        - libcublas-dev >={{ version }}
+    files:
+      - lib/lib*blas*_static.a
+      - targets/*/lib/lib*blas*_static.a
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcublas_static.a  # [unix]
+        - if not exist %LIBRARY_LIB%\\cublas.lib exit 1  # [win]
+
+about:
+  home: https://developer.nvidia.com/cublas
+  # A LicenceRef cannot be used in this case, see:
+  # https://github.com/conda-forge/staged-recipes/pull/12882#discussion_r504086944
+  license: LicenseRef-NVIDIA-End-User-License-Agreement
```

Comment:
I don't understand why this comment is here? We need the `LicenseRef-*` prefix to silent the linter error, so what's wrong with it?

Reply:
It's just referencing the earlier discussion where this came up and this was decided. Seems ok to keep

Reply:
Yes it needs to be reworded. It obviously *should* (not "cannot") be used.

Reply:
How about?

```suggestion
  # Uses `LicenceRef-*` per this discussion:
  # https://github.com/conda-forge/staged-recipes/pull/12882#discussion_r504086944
  license: LicenseRef-NVIDIA-End-User-License-Agreement
```

Reply:
LGTM

Reply:
Updated, it was copied verbatim from the `cudatoolkit` feedstock, so probably worth updating there as well.